src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/ResourceQueriesQueryString.feature.disabled

```diff
@@ -0,0 +1,105 @@
+Feature: Query String handling for GET requests for Resource Queries
+
+  Background:
+  
+
+  @ignore
+  Scenario: Ensure clients can GET information when querying by valid date
+    Given a school exists
+            | schoolId   | nameOfInstitution                             | gradeLevels               | educationOrganizationCategories          |
+            | 255901001 | Grand Bend High School                         | [ "Tenth grade" ]         | [ "School" ]                             |    
+    And there is a valid AcademicWeek using a valid BeginDate
+            | weekIdentifier | beginDate  | endDate    | totalInstructionalDays |
+            | string         | 2024-04-09 | 2024-04-09 | 0                      |
```

Comment:
"string" is a valid week identifier, but to another reviewer it might look very strange. Let's put something more realistic, like "Week One".

Reply:
Complete